packages/miniflare/src/plugins/kv/sites.ts

```diff
@@ -230,127 +97,61 @@ export async function getSitesNodeBindings(
     siteRegExps
   );
   return {
-    [BINDING_KV_NAMESPACE_SITE]: kProxyNodeBinding,
-    [BINDING_JSON_SITE_MANIFEST]: __STATIC_CONTENT_MANIFEST,
+    [SiteBindings.KV_NAMESPACE_SITE]: kProxyNodeBinding,
+    [SiteBindings.JSON_SITE_MANIFEST]: __STATIC_CONTENT_MANIFEST,
   };
 }
 
 export function maybeGetSitesManifestModule(
   bindings: Worker_Binding[]
 ): Worker_Module | undefined {
   for (const binding of bindings) {
-    if (binding.name === BINDING_JSON_SITE_MANIFEST) {
+    if (binding.name === SiteBindings.JSON_SITE_MANIFEST) {
       assert("json" in binding && binding.json !== undefined);
-      return { name: BINDING_JSON_SITE_MANIFEST, text: binding.json };
+      return { name: SiteBindings.JSON_SITE_MANIFEST, text: binding.json };
     }
   }
 }
 
-export function getSitesService(options: SitesOptions): Service {
+export function getSitesServices(options: SitesOptions): Service[] {
   // `siteRegExps` should've been set in `getSitesBindings()`, and `options`
   // should be the same object reference as before.
   const siteRegExps = sitesRegExpsCache.get(options);
   assert(siteRegExps !== undefined);
   // Ensure `siteRegExps` is JSON-serialisable
-  const serialisedSiteRegExps = {
-    include: siteRegExps.include && serialiseRegExps(siteRegExps.include),
-    exclude: siteRegExps.exclude && serialiseRegExps(siteRegExps.exclude),
-  };
+  const serialisedSiteRegExps = serialiseSiteRegExps(siteRegExps);
 
   // Use unsanitised file storage to ensure file names containing e.g. dots
   // resolve correctly.
   const persist = path.resolve(options.sitePath);
 
-  return {
+  const storageServiceName = `${SERVICE_NAMESPACE_SITE}:storage`;
+  const storageService: Service = {
+    name: storageServiceName,
+    disk: { path: persist, writable: true },
+  };
+  const namespaceService: Service = {
     name: SERVICE_NAMESPACE_SITE,
     worker: {
-      serviceWorkerScript: SCRIPT_SITE,
-      compatibilityDate: "2022-09-01",
+      compatibilityDate: "2023-07-24",
```

Comment:
I wonder if it would be good to put these dates into a constant rather than being dotted around the codebase?